yjit/src/codegen.rs

```diff
@@ -10521,6 +10521,13 @@ fn reg_method_codegen(klass: VALUE, mid_str: &str, gen_fn: MethodGenFn) {
     unsafe { METHOD_CODEGEN_TABLE.as_mut().unwrap().insert(method_serial, gen_fn); }
 }
 
+pub fn yjit_shutdown_free_codegen_table() {
+    unsafe {
+        drop(METHOD_CODEGEN_TABLE.as_mut());
+        METHOD_CODEGEN_TABLE = None;
```

Comment:
```suggestion
        METHOD_CODEGEN_TABLE = None;
```

nit: Probably plenty clear without using mem::drop in this context.

Reply:
😅 Clearly my Rust is not good enough.